tests/Node.py

```diff
@@ -1346,7 +1346,7 @@ def getNextCleanProductionCycle(self, trans):
     # TBD: make nodeId an internal property
     # pylint: disable=too-many-locals
     # If nodeosPath is equal to None, it will use the existing nodeos path
-    def relaunch(self, nodeId, chainArg=None, newChain=False, timeout=Utils.systemWaitTimeout, addOrSwapFlags=None, cachePopen=False, nodeosPath=None):
+    def relaunch(self, nodeId, chainArg=None, newChain=False, timeout=Utils.systemWaitTimeout, addSwapRemoveFlags=None, cachePopen=False, nodeosPath=None):
```

Comment:
I originally had done this because I was planning to also add a way to remove flags that were present, but must have lost those changes somewhere along the line, so I will revert these.